django_mongodb/compiler.py

```diff
@@ -153,9 +153,18 @@ def get_columns(self):
         columns = (
             self.get_default_columns(select_mask) if self.query.default_cols else self.query.select
         )
-        return tuple((column.target.column, column) for column in columns) + tuple(
-            self.query.annotations.items()
-        )
+        annotations_cnt = 0
+        result = []
+        for column in columns:
+            if hasattr(column, "target"):
+                target = column.target.column
+            else:
+                annotations_cnt += 1
+                target = f"annotation_{annotations_cnt}"
```

Comment:
I think we should make this name more "private" looking to lessen the chance it'll collide with the name of a user annotation. Something like `__annotation#`.